package.json

```diff
@@ -104,6 +104,7 @@
     "awesome-typescript-loader": "4.0.1",
     "connect-history-api-fallback": "^1.3.0",
     "copy-to-clipboard": "^3.0.8",
+    "copy-webpack-plugin": "^4.5.2",
```

Comment:
We need to copy `docs/src/public` to `docs/dist` on build.  Otherwise, the font files are not included.